core/drivers/pcap.cc

```diff
@@ -1,5 +1,8 @@
 #include "pcap.h"
 
+#include <algorithm>
```

Comment:
Maybe I am missing something, but the code change doesn't seem to need these headers here...